honeycomb-guide/src/project-structure/honeycomb-core.md

```diff
@@ -24,3 +25,8 @@ following:
 - **Orbit2**: Generic 2D implementation for orbit computation
 - **Vector2**: 2D vector representation
 - **Vertex2**: 2D vertex representation
+
+Note that optional features can be enabled when compiling this crate:
+
+- `utils` -- provides additionnal items that would qualify as quality-of-life, notably map generation functions.
```

Comment:
Not very clear from a user perspective.

Reply:
I rewrote this to be closer to the initial description of the feature